codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -409,6 +410,14 @@ class FluentClientGenerator(
                 }
             }
 
+            rust("/// Access the inner ${operationSymbol.name} builder as a reference.\n")
+            withBlockTemplate(
+                "pub fn inner(&self) -> &#{Inner} {", "}",
```

Comment:
What do you think about calling this `as_input()`? That should be less likely to name collide, and then people won't have to look at what the inner of the fluent builder is to understand what it's for.

Should we also add `to_input()` and `into_input()` for the owned return value side of things? The `to_input()` one may not be possible though unless the input is cloneable.

Reply:
Love it. Done.